packages/block-editor/src/components/text-alignment-control/README.md

```diff
@@ -0,0 +1,49 @@
+# TextAlignmentControl
+
+The `TextAlignmentControl` component is responsible for rendering a control element that allows users to select and apply text alignment options to blocks or elements in the Gutenberg editor. It provides an intuitive interface for aligning text with options such as `left`, `center` and `right`.
+
+## Usage
+
+Renders the Text Alignment Component with `left`, `center` and `right` alignment options.
+
+```jsx
+import { TextAlignmentControl } from '@wordpress/block-editor';
+
+const MyTextAlignmentControlComponent = () => (
+	<TextAlignmentControl
+		value={ textAlign }
+		onChange={ ( value ) => {
+			setAttributes( { textAlign: value } );
+		} }
+	/>
+);
+```
+
+## Props
+
+### `value`
+
+-   **Type:** `String`
+-   **Default:** `left`
+-   **Options:** `left`, `center`, `right`
```

Comment:
```suggestion
-   **Default:** `undefined`
-   **Options:** `left`, `center`, `right`, `justify`
```

- The default value is `undefined`, not `left`.
- Adds `justify` because the `Options` field actually needs to show all available options.